Packs/CortexXDR/Integrations/PaloAltoNetworks_XDR/PaloAltoNetworks_XDR.py

```diff
@@ -68,6 +68,11 @@ def clear_trailing_whitespace(res):
 
 
 class Client(BaseClient):
+
+    def __init__(self, base_url: str, headers: dict, timeout: int = 120, proxy: bool = False, verify: bool = False):
+        self.timeout = timeout
```

Comment:
why not pass the timeout to the `BaseClient` instead of passing it to each `htto_request` call?